include/material/newtonian.tcc

```diff
@@ -6,7 +6,10 @@ mpm::Newtonian<Tdim>::Newtonian(unsigned id, const Json& material_properties)
     density_ = material_properties.at("density").template get<double>();
     bulk_modulus_ =
         material_properties.at("bulk_modulus").template get<double>();
-    mu_ = material_properties.at("mu").template get<double>();
+    dynamic_viscosity_ =
+        material_properties.at("dynamic_viscosity").template get<double>();
+    incompressible_ =
```

Comment:
Is this a necessary material property then? I think it would be better to pass this from the projection method solver. Because if the user accidentally gives it as incompressible, the standard MPM solver cannot handle it. What do you think?

Reply:
I think passing a variable to a material requires a special function, which is public and needs to be defined in the base material class. I guess that's unnecessary. 

To compensate, I make the default value of `incompressible_` to be `false` and make a check at the Newtonian constructor to look whether the user defines incompressible bool or not in the input.

What do you think?

Reply:
See:
https://github.com/cb-geo/mpm/blob/073c703eaf8271df31993c9860a17e499244d03f/include/material/newtonian.tcc#L12-L15